docs/concepts/fields.md

```diff
@@ -828,3 +836,4 @@ print(User.model_json_schema())
 [frozen dataclass documentation]: https://docs.python.org/3/library/dataclasses.html#frozen-instances
 [Validate Assignment]: models.md#validate-assignment
 [Serialization]: serialization.md#model-and-field-level-include-and-exclude
+[Customizing JSON Schema]: json_schema.md#modifying-the-schema
```

Comment:
This link appears to be incorrect, I think this is the correct link

```suggestion
[Customizing JSON Schema]: json_schema.md/#customizing-json-schema
```

Reply:
Fixed with my most recent commit